build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
         mavenLocal()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.2'
+        classpath 'com.android.tools.build:gradle:2.3.3'
```

Comment:
It is generally a good idea to keep software updated; in the case of gradlew I hope for speedups and security fixes